src/Microsoft.AspNetCore.Hosting/Startup/StartupBase.cs

```diff
@@ -11,9 +11,47 @@ public abstract class StartupBase : IStartup
     {
         public abstract void Configure(IApplicationBuilder app);
 
-        public virtual IServiceProvider ConfigureServices(IServiceCollection services)
+        // REVIEW: This is a breaking change
+        public virtual void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        IServiceProvider IStartup.ConfigureServices(IServiceCollection services)
+        {
+            ConfigureServices(services);
+            return CreateServiceProvider(services);
+        }
+
+        protected virtual IServiceProvider CreateServiceProvider(IServiceCollection services)
```

Comment:
Support what used to work. I probably can't make this change anyways since it's breaking.